java-client/src/generated/java/org/opensearch/client/opensearch/dangling_indices/list_dangling_indices/DanglingIndex.java

```diff
@@ -138,15 +159,29 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
     /**
      * Builder for {@link DanglingIndex}.
      */
-
     public static class Builder extends ObjectBuilderBase implements ObjectBuilder<DanglingIndex> {
+        @Nullable
+        private String creationDate;
+        private Long creationDateMillis;
         private String indexName;
-
         private String indexUuid;
+        private List<String> nodeIds;
 
-        private String creationDateMillis;
+        /**
+         * API name: {@code creation_date}
+         */
+        public final Builder creationDate(@Nullable String value) {
```

Comment:
This is not in the spec, is it? Should it take a `LocalDateTime` (or have an override like that)?

Reply:
I'm adding it to the spec in https://github.com/opensearch-project/opensearch-api-specification/pull/462

Though now I think it probably shouldn't actually get added to the client as it's one of the "human-readable" fields that's conditionally returned based on `?human`, but we don't expose the `human` query parameter in the client